3.9/data-science-arango-rdf-adapter.md

```diff
@@ -35,6 +35,7 @@ Watch this
 [lunch & learn session](https://www.arangodb.com/resources/lunch-sessions/graph-beyond-lunch-break-2-11-arangordf/){:target="_blank"} to get an
 introduction on ArangoRDF - an RDF adapter developed with the community
 as a first step at bringing RDF graphs into ArangoDB.
+- Note: Since the recording of this session, ArangoRDF has been overhauled and extended. Package usage has changed, so please refer to the [Quickstart](#quickstart) section below for the latest specification.
```

Comment:
just want to highlight this comment (the lunch & learn session is now out of date) - happy to remove/adjust as needed!